source/proof-of-concept-guide/aws-infrastructure-monitoring.rst

```diff
@@ -52,7 +52,6 @@ Wazuh server
 
```

Comment:
As the `remove_from_bucket` setting line was removed, we need to update the emphasize-lines accordingly. 
![image](https://user-images.githubusercontent.com/61882981/216044426-025be7a8-1a31-472a-862b-b5747c8544cd.png)
The highlighted lines should be  `<name><AWS_BUCKET_NAME></name>` and `<aws_profile><AWS_PROFILE_NAME></aws_profile>`.